resolwe/flow/managers/dispatcher.py

```diff
@@ -324,6 +342,8 @@ def _prepare_data_dir(self, data_id):
         :return: The prepared data directory path.
         :rtype: str
         """
+        logger.debug(__("Preparing data directory for data id {}.", data_id))
```

Comment:
For consistency with listener's logging messages, use `logger.debug(__("Preparing data directory for Data with id {}.", data_id))` here.